lib/cli/interactive-setup/service.js

```diff
@@ -127,13 +141,56 @@ module.exports = {
       }
       projectName = await resolveProjectNameInput(context.options, workingDir);
       projectDir = join(workingDir, projectName);
-      await createFromTemplate(projectType, projectDir);
+      const templateUrl = `https://github.com/serverless/examples/tree/master/${projectType}`;
+      try {
+        process.stdout.write(`\n${chalk.green(`Downloading "${projectType}" template...`)}\n`);
+        await downloadTemplateFromRepo(templateUrl, projectType, projectName, { silent: true });
+      } catch (err) {
+        throw new ServerlessError(
+          'Could not download template. Ensure that you are using the latest version of Serverless Framework.',
+          'TEMPLATE_DOWNLOAD_FAILED'
+        );
+      }
+    }
+
+    let hasPackageJson = false;
+    try {
+      await fsp.access(join(projectDir, 'package.json'));
+      hasPackageJson = true;
+    } catch {
+      // pass
+    }
+
+    if (hasPackageJson) {
+      process.stdout.write(
+        `\n${chalk.green(`Installing dependencies with "npm" in "${projectName}" folder.`)}\n`
+      );
+      const npmCommand = await npmCommandDeferred;
+      try {
+        await spawn(npmCommand, ['install'], { cwd: projectDir });
+      } catch (err) {
+        if (err.code === 'ENOENT') {
+          process.stdout.write(
+            `\n${chalk.yellow(
+              'Cannot install dependencies as "npm" installation could not be found. Please install npm and run "npm install" in directory of created service.'
+            )}\n`
+          );
+        } else {
+          throw err;
```

Comment:
What do you think we should do in cases of other errors? Should we just re-throw them or try to wrap them in `ServerlessError`

Reply:
I think it's ok to stop with `ServerlessError`, as it's installation failure, which we're not sure if it's recoverable.